frame/elections-phragmen/src/lib.rs

```diff
@@ -153,6 +207,15 @@ pub trait WeightInfo {
 	fn remove_member_wrong_refund() -> Weight;
 }
 
+#[derive(Encode, Decode, Eq, PartialEq)]
+/// Storage version.
```

Comment:
```suggestion
/// Storage version.
#[derive(Encode, Decode, Eq, PartialEq)]
```

I think habit is that doc goes first then attributes, so that attributes don't get lost before big docs